src/vs/workbench/contrib/customEditor/browser/customEditorInputFactory.ts

```diff
@@ -185,6 +188,7 @@ export class ComplexCustomWorkingCopyEditorHandler extends Disposable implements
 			contentOptions: restoreWebviewContentOptions(backupData.webview.options),
 			state: backupData.webview.state,
 			extension,
+			codeWindow: getActiveWindow()
```

Comment:
I'm not confident with this one. It's potentially coming from backup, where `active` might not be the right container.